.circleci/config.yml

```diff
@@ -4,12 +4,11 @@ step-library:
   - &restore-cache
       restore_cache:
         keys:
-          - nav-cache-v5-{{ .Environment.CIRCLE_JOB }}-{{ checksum "Cartfile.resolved" }}
-          - nav-cache-v5-{{ .Environment.CIRCLE_JOB }} # used if checksum fails
```

Comment:
This line broke the cache for me. See https://app.circleci.com/pipelines/github/mapbox/mapbox-navigation-ios/3758/workflows/d308c9bd-af8a-4a23-8298-cbd9c24345d8/jobs/34125 run. 

@1ec5 Can you please clarify what it does and if I can remove it?

Reply:
This step restores the Carthage folder (with all the checkouts and build products) from a cache from the previous build. It’s based on a checksum of the resolved file, so that any time a dependency version changes, there’s a cache miss. It’s also possible to force a cache miss by incrementing the `v5` to a different version.

Reply:
As far as I see the cache with a key `- nav-cache-v5-{{ .Environment.CIRCLE_JOB }} # used if checksum fails` isn't saved anywhere.